Firestore/core/src/remote/bloom_filter.h

```diff
@@ -63,6 +82,14 @@ class BloomFilter final {
 
   // Bloom filter's bitmap.
   std::vector<uint8_t> bitmap_;
+
+  BloomFilter::Hash Md5HashDigest(const absl::string_view key) const;
```

Comment:
nit: Move these private method declarations to be `above` the private variable declarations. It is just convention in our code base to define the private functions _first_ then the private variables _last_.